go.mod

```diff
@@ -121,3 +121,5 @@ require (
 	google.golang.org/protobuf v1.34.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/madmin-go/v3 => github.com/vadmeste/madmin-go/v3 v3.0.0-20240812132301-2f1172f311fb
```

Comment:
Isn't it merged so we can get rid of this?

Reply:
yes, thanks for catching this